### ISSUE TITLE
Conference proceedings and Event proceedings in ORA data model

### ISSUE THREAD

**Issue by mrdsaunders** [tomwrobel/ora_data_model#84]
Hello there

The SE to ORA field mapping for `symplectic_field name="name-of-conference"` includes `if test="record is a Conference proceedings or Event proceedings"`.

What ORA type/subtype do `Conference proceedings` and `Event proceedings` equate to? I can't find them referred to in the `<work-type-mappings>` section of the document.   

Thanks

**Comment by jjpartridge** [comment]
The ora field name for "proceedings" is 'host_title'. 
In the data model (https://docs.google.com/spreadsheets/d/1SM5nJ-RPQX7_k6BKvBqQPnWPaR8xZ-h5aexTSIdhw3Q/edit#gid=1348472781) this field relates to the item types (and subtypes):

- Conference item (Abstract; Exhibition; Presentation; Poster; Addendum; Corrigendum; Erratum)
- Book section (Chapter; Introduction; Table of Contents; Abstract)
- Working paper (Discussion paper)
- Report (Confidential report)

This filed should not get confused with 'event_title' or 'event_series_title' 

Let me know if you need anything else.

**Comment by mrdsaunders** [comment]
Thanks Jason.  I read 'if test="record is a Conference proceedings or Event proceedings"` as referring to output type as opposed to a field name. My problem is that I don't know how to determine whether `record is a Conference proceedings or Event procedings.`

**Comment by mrdsaunders** [comment]
Thanks Jason. I read `if test="record is a Conference proceedings or Event proceedings"` as referring to output type as opposed to a field name. My problem is that I don't know how to determine whetherrecord is a Conference proceedings or Event procedings.`

**Comment by tomwrobel** [comment]
The relevant ORA type/subtypes are:

Conference item/any
Compositon/performance

Does this help?

Best wishes, and blue skies

Tom

--
Tom Wrobel
Software Engineer - BDLSS
thomas.wrobel@bodleian.ox.ac.uk / 01865 277608

________________________________________

**Comment by mrdsaunders** [comment]
Thanks. So to use ORA types/subtypes I can treat that conditional as `if test="record is a Conference item or Performance`.

Do we need to update the SE to ORA field mapping document with this sort of change??

**Comment by mrdsaunders** [comment]
We now have a conflict between the following as used for the SE 'name-of-conference' field:
 
1. 'Conference paper' - defined as `Conference item/no subtype` -> use `event title`
2. 'Conference proceedings' - defined as `Conference item/any` -> use `host title`

How do we define straight 'Conference' (in relevant section of SE to ORA field mapping, below) - since it takes ORA field `title` it will also conflict if it is defined as either above.

```
<symplectic_field name="name-of-conference">
    <if test="record is a Conference"><ora_field>title</ora_field></if>
    <if test="record is a Conference paper, or Conference poster"><ora_field>event_title</ora_field> 
     </if>
    <if test="record is a Conference proceedings or Event proceedings"> 
   <ora_field>host_title</ora_field>
    </if>
</symplectic_field>
```

**Comment by tomwrobel** [comment]
Good point, my apologies.

@jjpartridge do we still have Conference (i.e. the event itself) as a work type at all?

**Comment by jjpartridge** [comment]
No we don't have 'Conference' as an item type. All are mapped to 'Conference item'

**Comment by mrdsaunders** [comment]
Maybe:
<symplectic_field name=`name-of-conference`>
if work type is `Conference item/any` or `Composition/performance`:
    map to ORA `event title`?

**Comment by tomwrobel** [comment]
@mrdsaunders that sounds right to me. @jjpartridge ?

**Comment by jjpartridge** [comment]
Yes this looks right to me. Host would be if there was a proceedings outside of a journal publication (for example)

**Comment by mrdsaunders** [comment]
> Yes this looks right to me. Host would be if there was a proceedings outside of a journal publication (for example)

How would a proceedings outside of a journal publication be defined in the data?

**Comment by mrdsaunders** [comment]
Thanks. Will close once we agree how to maintain the mapping document.

**Comment by mrdsaunders** [comment]
Resolved at ORA-SE 17/10/2019. `Proceedings` is not a separate type from `Conference item` in ORA so do not need to map `host_title`.

**Comment by mrdsaunders** [comment]
```
<symplectic_field name=name-of-conference>
            <if test="record is Conference item/any or Composition/performance"><ora_field>event_title</ora_field></if>
</symplectic_field>
```
